shared-libs/user-management/src/users.js

```diff
@@ -342,15 +342,15 @@ const hasParent = (facility, id) => {
  */
 const mapUser = (user, setting, facilities) => {
   const facilityIds = Array.isArray(user.facility_id) ? user.facility_id : [user.facility_id];
-  const places = facilityIds.map(facility => getDoc(facility, facilities));
+  const places = facilityIds.filter(facilityId => facilityId).map(facility => getDoc(facility, facilities));
```

Comment:
I believe this is already covered by the changes in code I have done. Can you please rebase off of my branch. 
You should _not_ need to make changes to any api / shared libs code.